content/rest-api/index.textile

```diff
@@ -534,6 +534,14 @@ bc[json]. [{
   data: <optional clientData provided by the client>
 }]
 
+h3(#metadata-rest). Retrieve metadata for a channel
+
+<%= partial 'shared/_channel_metadata' %>
+
+h3(#enumeration-rest). Retrieve enumeration of all active channels
```

Comment:
This looks odd.  Why not just "Enumerate all active channels" instead of retrieve enumeration?